src/array.jl

```diff
@@ -753,14 +753,29 @@ end
 leveltype(::Type{T}) where {T <: CategoricalArray} = leveltype(nonmissingtype(eltype(T)))
 
 """
-    levels(x::CategoricalArray)
+    levels(x::CategoricalArray, skipmissing=true)
     levels(x::CategoricalValue)
```

Comment:
why for `CategoricalValue` you do not want to support `skipmissing`?

Reply:
For `CategoricalValue` we can never return `missing` as we only return the levels of the pool. Not sure whether passing `skipmissing` should be allowed nevertheless. Since `CategoricalValue` isn't a collection, it doesn't have to follow the definition from DataAPI. I'd tend not to allow it as I don't see in which cases you would call `levels` on an object which could be either an array or a `CategoricalValue`.

Reply:
OK - we can add it later if a good use is found.